### DISCUSSION TITLE
Same key being assigned to multiple subjects

### DISCUSSION THREAD

**Discussion by ajcrandall1** [olivierfriard/BORIS#715] [Q&A]
When coding a video - even when a subject is selected - I am seeing the window pop-up saying "the _ key code contains many behaviors. Choose one:". Each subject has unique codes as compared to the other subjects. Is there a way to prohibit other subject's codes from being activated when a single subject is selected/being coded for?

**Comment by olivierfriard** [comment]
I am sorry but I do not understand very well your question.

Since v. 8.24 it is possible to associate the same key to many subjects (and behaviors).

**Comment by ajcrandall1** [comment]
Hi,

My issue is that I have behaviors I am coding specific to each subject and do not want the option to have every behavior applied to each subject. When coding the video I would like to focus on one subject at a time, is it possible to “block” the behaviors for the other subject from being used?

Let me know if this makes more sense or I can try to explain another way
AJ

From: Olivier Friard ***@***.***>
Date: Tuesday, February 13, 2024 at 11:34 AM
To: olivierfriard/BORIS ***@***.***>
Cc: Crandall, AJ (NYSPI) ***@***.***>, Author ***@***.***>
Subject: Re: [olivierfriard/BORIS] Same key being assigned to multiple subjects (Discussion #715)
You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

ATTENTION: This email came from an external source. Do not open attachments or click on links from unknown senders or unexpected emails.


I am sorry but I do not understand very well your question.

Since v. 8.24 it is possible to associate the same key to many subjects (and behaviors).

**Comment by olivierfriard** [comment]
It is clear for me now.
It is not possible to restrict the usage of some behaviors to specific subjects.
bye
